### ISSUE TITLE
Invalid Alt Tags from Image Widget

### ISSUE THREAD

**Issue by Greg-Boggs** [pojome/elementor#5678]
When you insert an image with no alt tag, the HTML created by the image widget produces an invalid Alt tag. The invalid Alt tag does not meet wcag 2.0 accessibility guidelines. Alt tags for decroative, background, or redundant images should be:

```
alt=""
```

Meanwhile, the image widget produces.

```
alt
```

We are currently using the free version 2.1.4, although we do have a pro license which we haven't installed yet.

**Comment by bainternet** [comment]
@Greg-Boggs 

The Image widget uses the WordPress Native [`wp_get_attachment_image`](https://developer.wordpress.org/reference/functions/wp_get_attachment_image/) which returns an `alt` attribute correctly. (`alt=""`)

Check that the page source (view source) not the DevTools/inspector.

thanks.
